res/xml/latn_bepo_fr.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Inspired by the bépo layout, but adapted to smartphone -->
+<keyboard name="BEPO (Français)" script="latin">
+<row>
+    <key width="1.1" key0="b" key1="esc" key2="1" key4="|"/>
+    <key width="1.1" key0="é" key2="2" key4="è"/>
+    <key width="1.1" key0="p" key2="3"/>
+    <key width="1.1" key0="o" key2="4" key3="œ" key4="accent_circonflexe"/>
+    <key width="1.1" key0="v" key2="5" key4="{"/>
+    <key width="1.1" key0="d" key2="6" key4="}"/>
+    <key width="1.1" key0="l" key2="7" key4="("/>
+    <key width="1.1" key0="j" key2="8" key4=")"/>
+    <key width="1.1" key0="z" key2="9" key4="["/>
+    <key width="1.1" key0="w" key2="0" key4="]"/>
+  </row>
+  <row>
+    <key width="1.1" key0="a" key1="tab" key3="æ" key4="à"/>
+    <key width="1.1" key0="u" key1="accent_trema" key4="$"/>
+    <key width="1.1" key0="i" key4="&amp;"/>
+    <key width="1.1" key0="e" key1="€" key2="accent_aigu" key4="accent_grave"/>
+    <key width="1.1" key0="c" key4="ç"/>
+    <key width="1.1" key0="t" key1="²" key4=";"/>
```

Comment:
Do you use `²` so often ? It's currently in the number board with the `sup` modifier. Later, it will be very easy to type with https://github.com/Julow/Unexpected-Keyboard/issues/204

Reply:
quite often but that's probably not a typical use case